docs/getting-started/queries/gosdk/simple-data-query.md

```diff
@@ -1,8 +1,33 @@
 ---
 id: simple-data-query
-title: Query Data Using Keys
+title: Query a Data Using Keys
```

Comment:
To query data 
(plural) 
To query records
to query a record

Reply:
Done